ethergo/util/util.go

```
@@ -0,0 +1,17 @@
+package util
+
+import "strings"
+
+// FormatError applies custom formatting & noise reduction to error messages. Add more as needed.
+func FormatError(err error) string {
+	if err == nil {
+		return ""
+	}
+	errMsg := err.Error()
+
+	//if an error message contains embedded HTML (eg: many RPC errors), strip it out to reduce noise.
```

golangci-lint (1.62.2)

commentFormatting: put a space between `//` and comment text

(gocritic)